README.md

```diff
@@ -531,19 +538,89 @@ go get -u github.com/stripe/stripe-go/v79@v77.1.0-beta.1
 ```
 
 > **Note**
-> There can be breaking changes between beta versions. 
+> There can be breaking changes between beta versions.
 
 We highly recommend keeping an eye on when the beta feature you are interested in goes from beta to stable so that you can move from using a beta version of the SDK to the stable version.
 
 If your beta feature requires a `Stripe-Version` header to be sent, set the `stripe.APIVersion` field using the `stripe.AddBetaVersion` function to set it:
 
 > **Note**
-> The `APIVersion` can only be set in beta versions of the library. 
+> The `APIVersion` can only be set in beta versions of the library.
 
 ```go
 stripe.AddBetaVersion("feature_beta", "v3")
 ```
 
+### Raw Request
+
+If you would like to send a request to an API that is:
+
+- not yet supported in stripe-go (like any `/v2/...` endpoints), or
+- undocumented (like a private beta), or
+- public, but you prefer to bypass the method definitions in the library and specify your request details directly
+
+You can use the `rawrequest` package:
+
+```go
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/stripe/stripe-go/v79"
+	"github.com/stripe/stripe-go/v79/form"
+	"github.com/stripe/stripe-go/v79/rawrequest"
+)
+
+func make_raw_request() error {
+	stripe.Key = "sk_test_123"
+
+	payload := map[string]interface{}{
+		"event_name": "hotdogs_eaten",
+		"payload": map[string]string{
+			"value":              "123",
+			"stripe_customer_id": "cus_Quq8itmW58RMet",
+		},
+	}
+
+	// for a v2 request, json encode the payload
+	body, err := json.Marshal(payload)
+	if err != nil {
+		return err
+	}
+
+	v2_resp, err := rawrequest.Post("/v2/billing/meter_events", string(body), nil)
```

Comment:
do users need to specify Content-Type for this to work w/ v2 and JSON?

Reply:
nm, answered my question as I went along; we use the uri prefix (v1 or v2) to determine content type, right?

Reply:
correct, they can't specify API mode or content type manually. there didn't seem to be a point if we could just do it for them